src/components/SignUp/SignUp.tsx

```diff
@@ -1,5 +1,7 @@
 import React, { useState, useEffect } from 'react'
 import { useHistory } from 'react-router-dom'
+import { API_ROOT } from '../../constants/index'
```

Comment:
Now that we have these constants set up, we just have to toggle in the constants folder whether we're working locally or deploying.